integration/cli/build.configequality.test.ts

```diff
@@ -76,95 +74,104 @@ describe("build", () => {
           { source: "helm", getResources: getHelmResources },
         ];
 
-        it.each([
-          kind.ClusterRole,
-          kind.ServiceAccount,
-          kind.Secret,
-          kind.Service,
-          kind.Role,
-          kind.Deployment,
-          kind.MutatingWebhookConfiguration,
-          kind.ValidatingWebhookConfiguration,
-        ])("should have %s resources in both pepr and helm manifests", resourceKind => {
-          for (const { getResources } of sources) {
-            const res = resource.select(getResources(), resourceKind, peprUuid);
+        const resourceKinds = [
+          { kind: kind.ClusterRole, name: (uuid: string): string => uuid },
+          { kind: kind.ServiceAccount, name: (uuid: string): string => uuid },
+          { kind: kind.Secret, name: (uuid: string): string => `${uuid}-api-path` },
+          { kind: kind.Secret, name: (uuid: string): string => `${uuid}-tls` },
+          { kind: kind.Secret, name: (uuid: string): string => `${uuid}-module` },
+          { kind: kind.Service, name: (uuid: string): string => uuid },
+          { kind: kind.Service, name: (uuid: string): string => `${uuid}-watcher` },
+          { kind: kind.Role, name: (uuid: string): string => `${uuid}-store` },
+          { kind: kind.Deployment, name: (uuid: string): string => uuid },
+          { kind: kind.Deployment, name: (uuid: string): string => `${uuid}-watcher` },
+          { kind: kind.MutatingWebhookConfiguration, name: (uuid: string): string => uuid },
+          { kind: kind.ValidatingWebhookConfiguration, name: (uuid: string): string => uuid },
+        ];
+
+        it.each(
+          sources.flatMap(src =>
+            resourceKinds.map(r => ({
+              source: src.source,
+              getResources: src.getResources,
+              resourceKind: r.kind,
+              name: r.name,
```

Comment:
```suggestion
            resourceKinds.map(resourceKind => ({
              source: src.source,
              getResources: src.getResources,
              resourceKind: resourceKind.kind,
              name: resourceKind.name,
```

Not enforced by the linter, but I have a general preference against single-character variable names.